src/qml/qgismobileapp.qml

```diff
@@ -38,6 +38,40 @@ ApplicationWindow {
   property double sceneTopMargin: platformUtilities.sceneMargins(mainWindow)["top"]
   property double sceneBottomMargin: platformUtilities.sceneMargins(mainWindow)["bottom"]
 
+  Button {
+    text: "Tour"
+    anchors.centerIn: parent
+    z: 999999
+    onClicked: {
+      tour.open();
+    }
+  }
+
+  QFieldGuide {
+    id: tour
+    steps: [{
+        "title": qsTr("Zoom In Button"),
+        "description": qsTr("Main responsibility is described."),
+        "target": () => zoomInButton
+      }, {
+        "title": qsTr("Zoom Out Button"),
+        "description": qsTr("Main responsibility is described."),
+        "target": () => zoomOutButton
+      }, {
```

Comment:
As discussed in the chat, let's do one step for both zoom in and out buttons instead.

Reply:
completely removed